tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -56,20 +56,17 @@ def _get_normalized_size(size):
     return f'{size} {suffixes[suffix_index]}'
 
 
-def _get_vmware_datastore_summary_string(data_store_name=settings.vmware.datastore, vmware=None):
+@pytest.fixture
+def get_vmware_datastore_summary_string(vmware, vmwareclient):
     """Return the datastore string summary for data_store_name
 
     For "Local-Ironforge" datastore the string looks Like:
 
         "Local-Ironforge (free: 1.66 TB, prov: 2.29 TB, total: 2.72 TB)"
     """
-    system = VMWareSystem(
-        hostname=vmware.hostname,
-        username=settings.vmware.username,
-        password=settings.vmware.password,
-    )
+    data_store_name = settings.vmware.datastore
```

Comment:
why changing this to variable instead of positional argument?

Reply:
Right, no needed, removed